src/client/listjobs.js

```diff
@@ -0,0 +1,32 @@
+
+const server = "https://masteringjs-job-board.azurewebsites.net";
+async function getJobs() {
+  var jobs = await fetch(server+'/api/listjobs').then((res) => {
+    return res.json();
+  }).then((data) => {return data.jobs.splice(0,3)}).catch((err) => console.log(err));
+  console.log(jobs);
+
+  // write a for loop here and append to a single string with the
+  for(let i = 0; i < jobs.length; i++) {
+    document.getElementById('jobs').innerHTML += `
+        <a href="'/jobs/' + ${jobs[i]._id}">
```

Comment:
Nice. The only suggestion I have is to only set `innerHTML` once, because setting `innerHTML` tells the browser to re-render. Better to instead create a temp variable `let html = '';`, do `html += `<a href="...` here, and then do one `document.getElementById('jobs').innerHTML = html` assignment.